docs/INSTALL.md

```
@@ -0,0 +1,68 @@
+
+Ensure you have the following installed:
+
+- **Node.js** v16 or newer  
+- **npm** (bundled with Node.js)  
+- **PostgreSQL** v12 or newer   
+
+---
+
+## 1. Clone the repository
+
+git clone https://github.com/bluewave-labs/Headcount.git
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)